src/Compilers/Core/Portable/CommandLine/AnalyzerConfig.cs

```diff
@@ -21,7 +21,7 @@ public sealed partial class AnalyzerConfig
         // Matches EditorConfig section header such as "[*.{js,py}]", see https://editorconfig.org for details
         private static readonly Regex s_sectionMatcher = new Regex(@"^\s*\[(([^#;]|\\#|\\;)+)\]\s*([#;].*)?$", RegexOptions.Compiled);
         // Matches EditorConfig property such as "indent_style = space", see https://editorconfig.org for details
-        private static readonly Regex s_propertyMatcher = new Regex(@"^\s*([\w\.\-_]+)\s*[=:]\s*(.*?)\s*([#;].*)?$", RegexOptions.Compiled);
+        private static readonly Regex s_propertyMatcher = new Regex(@"^\s*([\w\.\-_]+)\s*[=:]\s*(.*?)\s*$", RegexOptions.Compiled);
```

Comment:
Should we be allowing whitespace within the key? 

eg: 
test value = allowed

Reply:
It appears that we do not currently support whitespace inside the key, but according to the specification it should be allowed:

> Key: The part before the first = (trimmed of whitespace, but including any whitespace in the middle).

Can you file a new issue for this?

Reply:
https://github.com/dotnet/roslyn/issues/71727